### PR TITLE
Boot from volume tutorial

### PR DESCRIPTION
**Pull request by swat30**
Instructions on how to boot an instance from a volume. Also includes two options for creating a bootable volume.

### CODE REVIEW

File: content/compute/boot-from-volume.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+/*
+Title: Booting an Instance from a Volume
+*/
```
Comment (thurloat) [inline content/compute/boot-from-volume.md:4]: does it need a sort? how does it appear in the menu?

Comment (swat30) [inline content/compute/boot-from-volume.md:4]: Possibly. Looks like this now:

image

File: content/compute/boot-from-volume.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
++--------------------------------------+--------------+------------------------------------------------------+
+| id                                   | name         | subnets                                              |
++--------------------------------------+--------------+------------------------------------------------------+
+| 7bc79d7b-0d04-488b-9513-b4695988840e | Demo Network | 28359fd5-5858-4ad3-94f4-673a31927603 10.200.100.0/24 |
++--------------------------------------+--------------+------------------------------------------------------+
+
+```
+
+In this example, we'll use the *4 GB* flavour and boot our instance on the *Demo Network*.
```
Comment (thurloat) [inline content/compute/boot-from-volume.md]: note that `boot-from-vol` is the name of your instance, and not an additional required argument.

File: content/compute/boot-from-volume.md (hunk 1 above)
Comment (thurloat) [inline content/compute/boot-from-volume.md:4]: IMO, Booting should be at the bottom of the list -- you can use `Sort: X` in the header meta -- the higher the sort value, the heavier the menu item weight to sink.

**Comment by swat30** [comment]
Updates as per your CR @thurloat :)

**Comment by thurloat** [comment]
LGTM! :poodle:

**Pull request merged**